workflows/phylorthology.nf

```diff
@@ -48,66 +70,416 @@ include { INPUT_CHECK } from '../subworkflows/local/input_check'
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC                      } from '../modules/nf-core/fastqc/main'
-include { MULTIQC                     } from '../modules/nf-core/multiqc/main'
-include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/custom/dumpsoftwareversions/main'
-
+include { BUSCO as BUSCO_SHALLOW                } from '../modules/nf-core/busco/main'
+include { BUSCO as BUSCO_BROAD                  } from '../modules/nf-core/busco/main'
+include { BUSCO                                 } from '../modules/nf-core/busco/main'
+include { DIAMOND_BLASTP as DIAMOND_BLASTP_TEST } from '../modules/nf-core/diamond/blastp/main'
+include { DIAMOND_BLASTP                        } from '../modules/nf-core/diamond/blastp/main'
+include { MAFFT                                 } from '../modules/nf-core/mafft/main'
+include { MAFFT as MAFFT_REMAINING              } from '../modules/nf-core/mafft/main'
+include { IQTREE                                } from '../modules/nf-core/iqtree/main'
+include { IQTREE as IQTREE_REMAINING            } from '../modules/nf-core/iqtree/main'
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     RUN MAIN WORKFLOW
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
+//def meta = ch_spp
+// This could be put into either the config file or specified via commandline. 
 
-// Info required for completion email and summary
-def multiqc_report = []
+// Function to get list of [ meta, [ file ] ]
+def create_og_channel(LinkedHashMap row) {
+    // create meta map
+    def meta  = [:]
+        meta.og   = row.orthogroup
+        meta.nspp = row.num_spp
+        meta.copy_num = row.mean_copy_num
+    // add path(s) of the OG file to the meta map
+    def og_meta = []
+        og_meta = [ meta, [ file(row.file) ] ] 
+    return og_meta
+}
 
 workflow PHYLORTHOLOGY {
-
+    
+    ch_inflation = ch_mcl_inflation.toList().flatten()
     ch_versions = Channel.empty()
+    
+    //
+    // MODULE: Prepare directory structure and fasta files according to 
+    //         OrthoFinder's preferred format for downstream MCL clustering
+    //
+    ORTHOFINDER_PREP (
```

Comment:
Think we talked about this, but it's definitely imaginable that the workflow could be run by specifying a single inflation parameter, skip the testing procedure, and go straight to the full analysis. So maybe those testing stages could be put into a subworkflow or something like that? 

The main thinking of having the MCL testing in the same workflow as the rest is so that the all-v-all blasting for the complete dataset (which could presumably be fairly time-consuming) can take place concurrently with the MCL testing component.